DSharpPlus/Entities/User/DiscordUser.cs

```diff
@@ -61,9 +62,23 @@ internal DiscordUser(TransportUser transport)
         [JsonProperty("username", NullValueHandling = NullValueHandling.Ignore)]
         public virtual string Username { get; internal set; }
 
+        /// <summary>
+        /// Gets this user's global display name.
+        /// </summary>
+        /// <remarks>
+        /// A global display name differs from a username in that it acts like a nickname, but is not specific to a single guild.
+        /// Nicknames in servers however still take precedence over global names, which take precedence over usernames.
+        /// </remarks>
+        [JsonProperty("global_name", NullValueHandling = NullValueHandling.Ignore)]
+        public virtual string GlobalName { get; internal set; }
+
         /// <summary>
         /// Gets the user's 4-digit discriminator.
         /// </summary>
+        /// <remarks>
+        /// As of May 15th, 2023, Discord has begun phasing out discriminators in favor of handles (@username); this property will return "0" for migrated accounts.
+        /// </remarks>
+        [Obsolete("Discriminators are being phased out in favor of handles. This property will be removed in a future version.")]
```

Comment:
this shouldn't be obsolete; bots remain on the old system for the foreseeable future

Reply:
I'd wager that discriminators are "technically" obsolete"; furthermore, bots rarely lookup other bots, so... I can remove it regardless.

Reply:
I use discrims in my code (kinda necessary to show them for users that still have them) and this caused it to throw obsolete warnings. Might be considered too early, maybe when rollout has finished (and when bots dont have them?)

Reply:
I'd obsolete once a vast majority of both users and bots have migrated over.